packages/components/src/ButtonGroup/ButtonGroup.module.css

```diff
@@ -0,0 +1,22 @@
+.buttonGroup {
+  @apply inline-grid;
+
+  grid-auto-flow: column;
```

Comment:
Nit: looks like Tailwind has `grid-flow-col`

Reply:
Thanks! I wish I had an extension for this. 🤔